access/handler.go

```diff
@@ -30,7 +33,7 @@ type HandlerOption func(*Handler)
 
 var _ access.AccessAPIServer = (*Handler)(nil)
 
-func NewHandler(api API, chain flow.Chain, finalizedHeader *synceng.FinalizedHeaderCache, me module.Local, options ...HandlerOption) *Handler {
```

Comment:
the emulator is currently using an older version of the api, with `func NewHandler(api API, chain flow.Chain)` as method signature.  I don't know why emulator integration test didn't catch this breakage